app/src/main/java/com/example/touchcar/presentation/main_menu/MainMenuFragment.kt

```diff
@@ -0,0 +1,93 @@
+package com.example.touchcar.presentation.main_menu
+
+import android.os.Build
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.touchcar.R
+import com.example.touchcar.domain.entity.Manufacturer
+import com.example.touchcar.presentation.main_menu.recycler.MainMenuAdapter
+import dagger.hilt.android.AndroidEntryPoint
+import java.util.*
+import javax.inject.Inject
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [MainMenu.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+
+@AndroidEntryPoint
+class MainMenu : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+    private lateinit var mainMenuAdapter: MainMenuAdapter
+    private lateinit var recyclerView: RecyclerView
+
+    @Inject
+    lateinit var mainMenuViewModel: MainMenuViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param1 = it.getString(ARG_PARAM1)
+            param2 = it.getString(ARG_PARAM2)
+        }
+    }
+
+    @RequiresApi(Build.VERSION_CODES.N)
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_main_menu, container, false)
+    }
+
+    override fun onViewCreated(itemView: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(itemView, savedInstanceState)
+
+        mainMenuViewModel.manufacturerLiveData
+            .observe(
+                this, androidx.lifecycle.Observer
+                { manufacturers ->
+                    mainMenuAdapter.updateViewHolder(manufacturers)
+                })
```

Comment:
Давай уберём именованный интерфейс отсюда и унесём лямбду за скобки:
```suggestion
        mainMenuViewModel.manufacturerLiveData.observe(this) { manufacturers ->
            mainMenuAdapter.updateViewHolder(manufacturers)
        }
```